### PR TITLE
Update aws-official-ami.md

### LINKED ISSUE #1886: AWS marketplace image doesn't honor the pg.passwd in server.conf
### Describe the bug

After spinning up an instance from the AWS marketplace image, I find I still need to use the default "quest" password to access the server through the pgsql interface. 

Also, what's the standard practice to change password? I don't find it in the doc. 

### To reproduce

_No response_

### Expected Behavior

_No response_

### Environment

```markdown
- **QuestDB version**:
- **OS**:
- **Browser**:
```

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Rajakavitha1**
fixes: https://github.com/questdb/questdb/issues/1886

### CODE REVIEW

File: docs/deployment/aws-official-ami.md [hunk 1]
```diff
@@ -35,8 +35,11 @@ category. To launch a QuestDB instance:
 An information panel displays the ID of the QuestDB instance with launch
 configuration details and hints for locating the instance in the EC2 console.
 
+The default `user` is `admin` and `password is `quest` to log in to the webconsole.
```
Comment (goodroot) [inline docs/deployment/aws-official-ami.md]:
```suggestion
The default user is `admin` and password is `quest` to log in to the webconsole.
```

File: docs/deployment/aws-official-ami.md [hunk 2]
```diff
@@ -35,8 +35,11 @@ category. To launch a QuestDB instance:
...
+
 ## QuestDB configuration
 
+You can connect to the instance where QuestDB is deployed using SSH.
```
Comment (goodroot) [inline docs/deployment/aws-official-ami.md]:
```suggestion
Connect to the instance where QuestDB is deployed using SSH.
```

File: docs/deployment/aws-official-ami.md [hunk 3]
```diff
@@ -68,6 +71,8 @@ during startup:
 pg.user=admin
 pg.password=...
 ```
+To use the credentials that are randomly generated and stored in the `server.conf`file, you
+need to restart the database using the command `sudo systemctl restart questdb`.
```
Comment (goodroot) [inline docs/deployment/aws-official-ami.md]:
```suggestion
To use the credentials that are randomly generated and stored in the `server.conf`file, restart the database using the command `sudo systemctl restart questdb`.
```

**Review by goodroot — COMMENTED** [review]
Hey @Rajakavitha1 ! 

Thank you very much.

Some minor comments.

**Comment by Rajakavitha1** [comment]
> Hey @Rajakavitha1 !
> 
> Thank you very much.
> 
> Some minor comments.

Done!!

**Comment by javier** [comment]
thanks!

**Pull request merged**